src/main/java/com/joejoe2/chat/models/GroupMessage.java

```diff
@@ -79,6 +82,31 @@ public static GroupMessage leaveMessage(GroupChannel channel, User actor, User s
     return message;
   }
 
+  private static GroupMessage banOrUnBanMessageFormat(
```

Comment:
Usually methods that do some common logic are put prior to the ones that are using it

Reply:
Ok

Reply:
it's just a cosmetic, not so crucial, just unusual I'd say.

Reply:
I have checked that it is already before banMessage#L97 and unbanMessage#L104. Are you saying about banOrUnBanMessageFormat ?

Reply:
Oh, yes, exactly, my bad :) All is good already